test/dish_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/dish'
+
+class DishTest < Minitest::Test
```

Comment:
These tests look awesome!  Do try to test other Dishes just to make sure this one isn't a fluke :)